### ISSUE TITLE
Tablet Issue with fractional scaling and xwayland apps (force_zero_scaling=true)

### ISSUE THREAD

**Issue by ooyjian** [hyprwm/Hyprland#3573]
### Hyprland Version

Hyprland, built from branch  at commit 1b48642fd15c433c53876f1b933dcd46265caf8f dirty (). Tag: v0.30.0

### Bug or Regression?

Bug

### Description

Hi all, 

I was trying a wacom intuos medium tablet on Hyprland (specifically with Aseprite and Krita, two apps that can only run with xwayland). On my display I have fractional scaling with 1.5, and on xwayland I enabled the force_zero_scaling setting. My problem is that the tablet can only reach 2/3 of the screen in those apps. If I set the 1.5 setting back to zero, this problem disappears. 

Couldn't find someone else with similar problems, so I figure this is the best place to ask. Not sure if this is the issue with Hyprland or issues with xwayland apps. 

Thanks!

Related Configs:
```
monitor=HDMI-A-1, 2560x1440,1920x0,1.5
monitor=eDP-1, preferred, auto, 1.75

xwayland {
	force_zero_scaling = true
}

...

device:wacom-intuos-bt-m-pen { 
	transform=2560x1440
	output=HDMI-A-1
}
```

### How to reproduce

1. Set scaling to be 1.5 on monitor
2. Set force_zero_scaling=true for xwayland
3. Use tablet in Krita

**Comment by ooyjian** [comment]
Any suggestions?

**Comment by Ant01n3** [comment]
Hello,

Same problem here with v 0.30.0 (and later built from git). It seems mouse events are correctly scaled back, but not tablet and touch events. This option would indeed be so great for Krita that is far from having Wayland support actually.

**Comment by hanspaerna** [comment]
Having the same problem.

**Comment by vaxerski** [comment]
MRs welcome. I do not use tablets.

**Comment by Ant01n3** [comment]
> MRs welcome. I do not use tablets. Typing "I have the same problem" will not speed this up.

Sorry, I just wanted to add that touch was also affected. In the meantime, I looked at the problem and I think I may have a solution: mouse events coordinates are scaled by `m_fX11SurfaceScaledBy` but tablet or touch events are not (in managers/input/Tablets.cpp and Touch.cpp). 

I tried to run my modified version of Hyrpland this afternoon and had no problems, however I do not know the code base at all, and I may also have forgotten a lot of things... Should I try to make a pull request? 

In any case, I'm available to test things if needed with my own tablet.
